src/main/java/org/broadinstitute/hellbender/utils/gcs/BucketUtils.java

```diff
@@ -81,6 +81,27 @@ public static String makeFilePathAbsolute(String path){
         }
     }
 
+    /**
+     * Appends path to the given dir/folder.
+     * @param dir the folder to append the path to
+     * @param path the path relative to dir
+     * @return the appended path as a String.
+     */
+    public static String appendPathToDir(String dir, String path) {
```

Comment:
Can you convert the String to a `Path`, and use `resolve()` instead of adding this method?

Reply:
Sure, will do.